README.md

```diff
@@ -24,12 +24,12 @@ User subscriptions are managed through a table
 
 ```sql
 create table realtime.subscription (
-    id bigint not null generated always as identity,
-    user_id uuid not null,
+    id uuid primary key,
     entity regclass not null,
-    filters realtime.user_defined_filter[],
-    created_at timestamp not null default timezone('utc', now()),
-    constraint pk_subscription primary key (id)
+    filters realtime.user_defined_filter[] not null default '{}',
```

Comment:
@olirice perhaps it's time to rename the type from `realtime.user_defined_filter` to something like `realtime.subscription_defined_filter`

Reply:
`user_defiend_filter` still seems right to me or `subscription_filter` could also work. If you have a preference lmk and i'll update